primitives/transaction/src/account/staking_contract/structs.rs

```diff
@@ -18,13 +18,16 @@ use crate::{SignatureProof, Transaction, TransactionError};
 ///         * Create
 ///         * Update
 ///         * AddStake
-///     The type of transaction, parameters and proof are given in the data field of the transaction.
+///
+/// The type of transaction, parameters and proof are given in the data field of the transaction.
```

Comment:
This fix seems incorrect(?). It looks like it should belong inside the `1.`.

Reply:
It's part of `1.` but being on the next line directly after `* AddStake` it thinks its part of the list thus needs to be indented up to the same level.

Reply:
Before: ![image](https://github.com/user-attachments/assets/885f5949-0a2a-401e-9c21-ec342cdf2515).
After: ![image](https://github.com/user-attachments/assets/d13fbf82-39da-4741-9a10-9ef4317490e1).

Both are incorrect, as far as I can tell.

Before, it was part of `* AddStake` which is clearly incorrect. Afterwards, it's now a free-standing text, but it should be part of the first enumeration item "1.".

I don't know if that can be fixed properly.

If not, perhaps we could reword the text so that

> The type of transaction, parameters and proof are given in the data field of the transaction.

can right after the "1."?

Reply:
Addressed